src/main/java/analysis/GoblintAnalysis.java

```diff
@@ -111,10 +125,29 @@ private void preAnalyse() {
     }
 
 
+    private void abortAnalysis() {
+        if (lastAnalysisTask != null && !lastAnalysisTask.isDone()) {
+            Process goblintProcess = goblintServer.getGoblintRunProcess().getProcess();
+            int pid = Math.toIntExact(goblintProcess.pid());
+            UnixProcess unixProcess = new UnixProcess(pid);
+            try {
+                unixProcess.kill(SIGINT);
+                lastAnalysisTask.cancel(false);
```

Comment:
Does this `cancel` really do anything? According to the `false` argument, it doesn't try to interrupt the task.